src/igvfd/schemas/construct_library.json

```diff
@@ -140,14 +140,14 @@
                 "genome-wide"
             ]
         },
-        "origins": {
-            "title": "Origins",
+        "selection_criteria": {
+            "title": "Selection Criteria",
             "description": "The criteria used to select the sequence material cloned into the library.",
             "type": "array",
             "minItems": 1,
             "uniqueItems": true,
             "items": {
-                "title": "Origin",
+                "title": "Selection Criteria",
```

Comment:
criterion rather than criteria

Reply:
singular item, vs plural name for a list

Reply:
Title updated.